include/net/loopback.h

```diff
@@ -0,0 +1,31 @@
+/** @file
+ * @brief Loopback control interface
+ */
+
+/*
+ * Copyright (c) 2017 Intel Corporation
```

Comment:
2022. Please add yourself in the copyright.

Reply:
You're totally right, lazy copy paste action